### ISSUE TITLE
Courseplay file isn't save with the game data. 

### ISSUE THREAD

**Issue by VardemisDatara** [Courseplay/courseplay#1231]
Hello, 
I have FS2015 on steam, I have arround 125 mods (map mods, machine, etc). 
Before the last big update of FS it was working properly nothing to say but after the update CP started not behave normaly. 
Each time I launch a savegame CP ask to scan the field and If I want to pay the workers. 
All the course I made disappeared in game. 
I tried with mods and without it's the same. 
I tried without maps mods (compost soil, multifruit, etc) thinking they were messing around some of the savefile, but it was still the same. 
I look up backup files and it appears that the file containing the courseplay data for the field isn't created or isn't saved in the directory. 

I think that the course saved in the course file are linked somehow the data in the field file so that why course are still in the file but not appearing in the game (if the course file is saved as it should, sometimes it doesn't). 

I take a look at the property of the save directory and it seems to be in "read only". 
I change that and it doesn't change anything. I will try to put all the directory tree in writing mode next time maybe it will help. 

I've tried on new game also it the same behavior. 

I've tried the solution of the  #1155 issue and it doesn't work. 

Can you help me identified the problem ? 

technical specification : 
Win7 and WIN10
FS2015 uptodate on steam
CP uptodate

**Comment by VAHA022rus** [comment]
Remove all fashion and check

18.09.2015, 19:09, "VardemisDatara" <notifications@github.com>:

**Comment by VardemisDatara** [comment]
fashion ?

**Comment by VAHA022rus** [comment]
I understand but I still can

18.09.2015, 20:39, "VardemisDatara" <notifications@github.com>:

**Comment by VardemisDatara** [comment]
I really didn't understand what you mean :-s :(

**Comment by VAHA022rus** [comment]
I mean the fact that you asked)

**Comment by VAHA022rus** [comment]
Just throw all mods, and then add and look for problems

**Comment by VardemisDatara** [comment]
Already did it ;)

**Comment by VAHA022rus** [comment]
Spasito for understanding :)

**Comment by Raints** [comment]
First post your log, then we might help you.

**Comment by VAHA022rus** [comment]
:Yes, he could not if he has a problem in the modes

**Comment by EnricoDutch** [comment]
I have the same issue.
And I think the problem came with the latest FS update.
https://github.com/Courseplay/courseplay/files/39820/log.txt

**Comment by Raints** [comment]
Not sure where to start. 

First things first. 
Remove mods:
Cat_Challenger_MT955C_UNZIP      Unzip it, then add.
Cat_Fleet_15_UNZIP_ME    Unzip it, then add.
Class_avero220_C6_pack    
Gilibert_Pack_A_EXTRAIRE    Unzip again
Kverneland_Wrapper1            Fs2013 mod
Lely_Tornado1zu1                  Fs2013 mod
Liebherr_L576_Pack_Extract_Files          Unzip.
ls15mod_installerV1.0.0.4         Not even sure what that is.
MapDoorTrigger                     Probably Fs2013 mod

First try to remove  all those errors, then we should look further.

**Comment by EnricoDutch** [comment]
I've done what you asked. I even reinstalled the game without the last update. But it goes wrong, if you quit the current game. Then it seems to refresh the "courseplay.xml" in the savegame folder. There is a backup. if you put that in your savegame, than you have your courseplay settings as they are supposed to.

**Comment by Raints** [comment]
Maybe you can show log again now. Try without  MachineryRental mod. Not sure anymore but i had some problems with it.

Btw. Try only one modded map in your mod folder.

**Comment by FarmerFiero** [comment]
Hello all
I just had this problem too.   Courseplay couldn't read the .Xml and was throwing the "error reading end tag" in the log.   After digging in the .Xml and comparing it to a working .Xml from another map, I noticed that a few lines of code were missing...  I added the missing code and deleted some of my largest courses, and it worked.    Im currently on the Michigan Cash crops map and my courses were just too large to load up.    The lines of code i was missing were just below the last course at the very bottom of the .Xml.    Below is what it should look like.  For some reason or another when saving REALLY large courses, the last 3 lines of code didn't get written. 
      
            <waypoint412 pos="-1070.65 1411.27" angle="-90.70" speed="0" wait="1" />
            <waypoint413 pos="-1070.65 1411.27" angle="-90.69" speed="0" crossing="1" />
            </course>
     </courses>
</XML>

I hope this helps for anybody else having a similar issue.
